tests/stub/authorization/verify_authentication/scripts/v5x0/router.script

```diff
@@ -1,7 +1,7 @@
 !: BOLT #VERSION#
 !: ALLOW RESTART
 
-A: HELLO {"user_agent": "*", "[routing]": {"{}": "*"}, "scheme": "basic", "principal": {"U": "*"}, "credentials": {"U": "*"}, "[realm]": ""}
```

Comment:
Is this really needed? The router shouldn't be used in the test if there's no routing going on. And if there is, I'd expect the routing context to be not `null`.

(same for the other router script)